sdb/commands/stacks.py

```diff
@@ -190,6 +211,17 @@ def task_struct_get_state(task: drgn.Object) -> str:
         exit_state = task.exit_state.value_()
         return Stacks.TASK_STATES[(state | exit_state) & 0x7f]
 
+    #
+    # Resolve TSTATE shortcut and/or sanitize it to standard uppercase
+    # notation if it exists.
+    #
+    @staticmethod
+    def resolve_state(tstate: str) -> str:
+        tstate = tstate.upper()
```

Comment:
Besides the other place mentioned below where `self.args.tstate` is set directly to whatever is returned by this function, it also seems like this function would mutate its arguments anyway with this statement.

Reply:
I think that in python, strings are constant, so `tstate.upper()` doesn't modify `tstate`, and `tstate = ...` doesn't modify the caller's variable, just reassigns the local variable to a different value.